static/js/components/util/customdatetime.vue

```diff
@@ -70,8 +70,7 @@
     data: function() {
       return {
         tooltipConfig: tooltipConfig,
-        datetimeFormat: datetimeFormat,
-        theValue: this.value
```

Comment:
Is value an internal field so you don't need this?

Reply:
Yep, it's a prop. If you're wondering how there is a `value` prop being passed into the `customdatetime` component, it's from using `v-model`: https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components